src/plugins/packages.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+const fs = require("fs");
+const colors = require("colors/safe");
+const Helper = require("../helper");
+const themes = require("./themes");
+
+const stylesheets = [];
+
+module.exports = {
+	getStylesheets: getStylesheets,
+	loadPackages: loadPackages,
+};
+
+const packageApi = function(packageName) {
+	return {
+		addStylesheet: addStylesheet.bind(this, packageName),
+	};
+};
+
+function addStylesheet(packageName, filename) {
+	stylesheets.push(packageName + "/" + filename);
+}
+
+function getStylesheets() {
+	return stylesheets;
+}
+
+function loadPackages() {
+	fs.readdir(Helper.getPackagesPath(), (err, packages) => {
+		if (err) {
+			return;
+		}
+		packages
```

Comment:
unnecessary newline

Reply:
Also store the a map of loaded packages

Reply:
For what purpose? What's the reason for storing them until we use it for something?

Reply:
Verifying that requested package exists, see comment down below.

Reply:
Ah right, I just saw that. Makes sense.

> Also will need it to grab manifest.json as we discussed.

Sure, but this isn't in this PR, so it wouldn't be a reason for storing them yet. But yeah, I'll do it for verifying.